apis/rust/node/Cargo.toml

```diff
@@ -26,6 +26,8 @@ futures-concurrency = "7.3.0"
 futures-timer = "3.0.2"
 dora-arrow-convert = { workspace = true }
 aligned-vec = "0.5.0"
+communication-layer-request-reply = { workspace = true }
```

Comment:
Nodes should only communicate through the daemon, not with the coordinator. Otherwise things will get very complex in non-standard deployments.